demo/api/src/db/migrations/Migration20241016123307.ts

```diff
@@ -0,0 +1,9 @@
+import { Migration } from "@mikro-orm/migrations";
+
+export class Migration20241016123307 extends Migration {
+    async up(): Promise<void> {
+        this.addSql(
+            'create table "BrevoConfig" ("id" uuid not null, "createdAt" timestamp with time zone not null, "updatedAt" timestamp with time zone not null, "scope_domain" text not null, "scope_language" text not null, "senderMail" text not null, "senderName" text not null, constraint "BrevoConfig_pkey" primary key ("id"));',
+        );
+    }
+}
```

Comment:
This migration should be included in the module's migrations

Reply:
Corrected here: 2540ea3d97fda5e54b154c93cd847858b05eea85